### ISSUE TITLE
Some problems on reproducing

### ISSUE THREAD

**Issue by PTL2011** [Academic-Hammer/SciTSR#17]
Hi, thanks for your great work! 
During reproducing this work, I have encountered several problems. First, since there was no "rel" dir in the test data dir, so I just tried to produce it with rel_gen.py you provided, during which a lot "no match" errors were printed. Second, assertion error that "test_path/json" is not dir occurred when running the train.py. Then I just change the "with_cells" parameter to False when creating test dataset. 
So I am wondering whether I was doing right and were there some procedures I missed? And, what is the difference between with_cells=True and with_cells=False? Btw, does the "json" dir in your code mostly mean the "structure" dir containing json files?  Please give me some hints.
Thanks again!

**Comment by CZWin32768** [comment]
Yes, the "json" dir in our code means the "structure" dir in the released SciTSR dataset. 

I rename this folder from "json" to "structure" to make it more meaningful, but I forgot to update the related code at [here](https://github.com/Academic-Hammer/SciTSR/blob/79954b5143295162ceaf7e9d9af918a29fe12f55/scitsr/data/loader.py#L111).

A walkaround is to rename the "structure" folder to "json".

**Comment by CZWin32768** [comment]
`with_cells` means to load structure labels.

The model is trained on the constructed graphs, so it is ok to set `with_cells` as `False`.
During the evaluation, `with_cells` should be set as `True` to load structure labels.

**Comment by Pyten** [comment]
@CZWin32768 Thanks for your detailed replay. During reproducing your work, I got 98%acc after 15 epochs, is this similar to your result? And if I want to evaluate Precision and Recall during training, how should I do it?  I noticed there is eval_relations API for evaluation, but can I directly use it with label and model output? Please offer some tips. Thanks again!

**Comment by akshowhini** [comment]
@PTL2011 Would you mind sharing your work on achieving 98%. Mine was struck at 68% with no betterment after 8 epochs. Also, the model from my training is not at all finding vertical relations across objects, and the predicted horizontal relations are no good.

**Comment by akshowhini** [comment]
@PTL2011 

The accuracy from the test epochs isn't a great deal, because the even at 2000 training dataset, and 2 epochs, more than 95% accuracy is shown. **What was the loss?** 

Mine was struck at 88% with no betterment after 4 epochs. Also, the model from my training is not at all predicting vertical relations across objects, and the horizontal relation prediction are no good either.

@CZWin32768  Could you please spare sometime providing some tips here. TIA

**Comment by kbrajwani** [comment]
@CZWin32768 @Pyten  hey can you help me to inference image through trained model. I have trained model now i have new image so what steps i can perform to get the table structure.
